gui/src/mainwindow.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>584</width>
-    <height>697</height>
+    <width>749</width>
+    <height>849</height>
```

Comment:
We should leave this narrower width for users with small screens.

Reply:
I am not comfortable with the way I added these items to the gui -- break the layout, add some items, then put in grid layout.

I will spend some time trying to undo as many of the changes as I can (in the xml directly) -- not sure what is best workflow here (the break layout seems pretty heavy handed)

Reply:
It can be difficult. I have had to hand edit in the past but I think you really need to do is expand the channel selector over one so it's spanning the whole row. When your editing the window just leave the width the minimum possible that everything fits.

Reply:
> Question, this additional field (alertch) seems extensible, e.g. it should just work if someone updates firmware but doesn't change the saved settings. But I'm not sure that upgrade pattern worked. I'll need to test that case a bit more.

Yes it should just work if you update. On boot it is set default, if there is no JSON command saved in the flash it will stay default. I was trying to keep as much back/forwards compatibility as possible. So even an Old/Newer gui shouldn't crash at least. The way you did it works! The most recent Gryo changes I did on the other hand will cause some backwards issues.

> I am looking more at the mechanisms for sending data back to the unit (sort of a master/slave bidirectional thing, or s-bus, etc. This feels like an OpenTX challenge, so I will dig a bit.

Let me know if you want some info here. I've been working on https://github.com/EdgeTX/edgetx/pull/1374 and really need to get this one finished first. Wish I had more free time. the last few months have been crazy. 

The real challenge is that the current Frsky modules aren't bidirectional. Now that serial refactor is complete in ETX I would like to get back to working on the ESP as a open source option for communication bidirectionally.

Reply:
Ok, a bit of hand edit and cleanup and the changes are easier to see in change review.

I'll push an update and then let me look at the other mechanisms, including your changes above.

I'm waiting on the in circuit debugger to arrive, plus it looks like I better plan a migrate to EdgeTx :-/